app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,276 @@
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Proxima Nova', sans-serif;
+}
+
+body {
+  background-color: #f5f6f8;
+}
+
+h2,
+h3,
+h5 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+h1 {
+  text-align: center;
+}
+
+.text-align-center {
+  text-align: center;
+  align-self: center;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.first-field {
+  margin-top: 5px;
+}
+
+.new_user input {
+  border: none;
+  border-radius: 0;
+}
+
+.new-category {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  width: 100%;
+  justify-content: space-between;
+  flex: 1;
+}
+
+.new-category input {
+  border: none;
+  border-radius: 0;
+}
+
+.save-catagory input {
+  width: auto !important;
+  height: auto !important;
+  padding: 10px;
+  border: 1px solid white;
+}
+
+.new-category div input {
+  width: 100%;
+  height: 40px;
+}
+
+.new-category div select {
+  width: 100%;
+  height: 40px;
+}
+
+.grey-color {
+  color: var(--grey);
+}
+
+.light-grey-color {
+  color: #c3c3c3;
+  font-weight: lighter;
+}
+
+.field {
+  border-top: rgb(232, 232, 232) 1px solid;
+}
+
+.field:last-child,
+.last-field {
+  border-bottom: rgb(232, 232, 232) 1px solid;
+}
+
+:root {
+  --blue: #3778c2;
+  --green: #5fb523;
+  --grey: #434b54;
+  --light-grey: #efefef;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+  text-align: center;
+}
+
+.blue-button {
+  display: inline;
+  background-color: var(--blue);
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 5px;
+  font-size: 1.2rem;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.grey-button {
+  color: var(--grey);
+  border: none;
+  padding: 10px 20px;
+  border-radius: 5px;
+  font-size: 1.2rem;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.login-signup-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+  align-self: center;
+  width: 100%;
+  max-width: 500px;
+}
+
+.home-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: end;
+  gap: 150px;
+}
+
+.home-container h1 {
+  font-size: 2rem;
+  text-align: center;
+  font-weight: 700;
+  color: var(--grey);
+}
+
+.navbar-container {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  background-color: var(--blue);
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.navbar-container i {
+  font-size: 1.5rem;
+}
+
+.white-color {
+  color: white !important;
```

Comment:
- [ ] Kindly don't use `!important`. If you use important too liberally, it can make it harder to override styles later on. This can create a situation where you need to use important even more, leading to a vicious cycle that makes your CSS harder to manage.